### PR TITLE
(chore) Update to new Vimeo SDK

### PR DESCRIPTION
**Pull request by nitriques**
The old one is sometimes broken or lacks features

### CODE REVIEW

File: src/modules/oembed-vm.js [hunk 1]
```diff
@@ -21,62 +21,77 @@
...
+				App.loaded(VM, function (Player) {
+					var player = new Player(container.get(0), {
+						id: id,
+						background: false,
+						autopause: true,
+						autoplay: options.autoplay,
+						byline: false,
+						playsinline: true,
+						controls: true,
+						loop: options.loop,
```
Comment (fhamon) [inline src/modules/oembed-vm.js]: They should be settable by data attributes too.

Comment (fhamon) [reply src/modules/oembed-vm.js]: and fallback to the options.* settings.

Comment (nitriques) [reply src/modules/oembed-vm.js]: Yes. This is out of scope tho.

Comment (fhamon) [reply src/modules/oembed-vm.js]: You are more than welcome to send another pr then ;) haha

**Comment by nitriques** [comment]
@fhamon wanna approve please ? The red button is scaring me.

**Pull request merged**